src/app/feedback/feedback-view.scss

```diff
@@ -38,12 +34,3 @@
     width: 100%;
   }
 }
-
-@media (min-width: $screen-md) {
-  .title-edit {
-    width: 40%;
-  }
-  .view-container {
-    padding: 3vh 25vw;
-  }
-}
```

Comment:
Hi @lmmrssa, seems if to delete media, the chat will be too wide on a large screen. You don't think so?